mkosi

```diff
@@ -429,7 +433,7 @@ def install_debian_or_ubuntu(args, workspace, run_build_script, mirror):
     cmdline = ["debootstrap",
                "--verbose",
                "--variant=minbase",
-               "--include=systemd",
+               "--include=systemd,usrmerge",
```

Comment:
@fsateler After addeding the `usermerge` package, I'm not able to build the latest Debian 8 image:

```bash
./mkosi -d debian -r jessie -t directory -o debian-test
```

(There's no `mkosi.default` in the working directory). Error message is then:

```bash
DISTRIBUTION:
          Distribution: debian
               Release: jessie
                Mirror: http://httpredir.debian.org/debian

OUTPUT:
         Output Format: directory
                Output: /home/stefan/Repositories/mkosi/debian-test
       Output Checksum: n/a
      Output Signature: n/a
Output nspawn Settings: n/a

PACKAGES:
              Packages: none
           Extra Trees: none
          Build Script: none
         Build Sources: /home/stefan/Repositories/mkosi
        Build Packages: none
       nspawn Settings: none
‣ Detaching namespace...
‣ Detaching namespace complete.
‣ Setting up package cache...
‣ Setting up package cache /home/stefan/Repositories/mkosi/.mkosi-afg8cik0 completed.
‣ Setting up temporary workspace.
‣ Temporary workspace in /home/stefan/Repositories/mkosi/.mkosi-swlyajv3 is now set up.
‣ Setting up basic OS tree...
‣ Setting up basic OS tree completed.
‣ Installing Debian...
W: Cannot check Release signature; keyring file not available /usr/share/keyrings/debian-archive-keyring.gpg
I: Retrieving Release 
I: Retrieving Packages 
I: Validating Packages 
I: Resolving dependencies of required packages...
I: Resolving dependencies of base packages...
I: Found additional required dependencies: acl adduser dmsetup insserv libaudit-common libaudit1 libbz2-1.0 libcap2 libcap2-bin libcryptsetup4 libdb5.3 libdebconfclient0 libdevmapper1.02.1 libgcrypt20 libgpg-error0 libkmod2 libncursesw5 libprocps3 libsemanage-common libsemanage1 libslang2 libsystemd0 libudev1 libustr-1.0-1 procps systemd systemd-sysv udev 
I: Found additional base dependencies: debian-archive-keyring gnupg gpgv libapt-pkg4.12 libreadline6 libstdc++6 libusb-0.1-4 readline-common 
I: Checking component main on http://httpredir.debian.org/debian...
E: Couldn't find these debs: usrmerge
Traceback (most recent call last):
  File "./mkosi", line 1554, in <module>
    main()
  File "./mkosi", line 1550, in main
    build_stuff(args)
  File "./mkosi", line 1511, in build_stuff
    (raw, tar) = build_image(args, workspace, run_build_script=False)
  File "./mkosi", line 1448, in build_image
    install_distribution(args, workspace.name, run_build_script)
  File "./mkosi", line 587, in install_distribution
    install[args.distribution](args, workspace, run_build_script)
  File "./mkosi", line 498, in install_debian
    install_debian_or_ubuntu(args, workspace, run_build_script, args.mirror)
  File "./mkosi", line 472, in install_debian_or_ubuntu
    subprocess.run(cmdline, check=True)
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['debootstrap', '--verbose', '--variant=minbase', '--include=systemd,usrmerge', '--exclude=sysv-rc,initscripts,startpar,lsb-base,insserv', 'jessie', '/home/stefan/Repositories/mkosi/.mkosi-swlyajv3/root', 'http://httpredir.debian.org/debian']' returned non-zero exit status 1
Traceback (most recent call last):
  File "/usr/lib/python3.5/weakref.py", line 595, in _exitfunc
    f()
  File "/usr/lib/python3.5/weakref.py", line 519, in __call__
    return info.func(*info.args, **(info.kwargs or {}))
  File "/usr/lib/python3.5/tempfile.py", line 797, in _cleanup
    _shutil.rmtree(name)
  File "/usr/lib/python3.5/shutil.py", line 474, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/usr/lib/python3.5/shutil.py", line 412, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/usr/lib/python3.5/shutil.py", line 412, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/usr/lib/python3.5/shutil.py", line 412, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/usr/lib/python3.5/shutil.py", line 412, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/usr/lib/python3.5/shutil.py", line 416, in _rmtree_safe_fd
    onerror(os.rmdir, fullname, sys.exc_info())
  File "/usr/lib/python3.5/shutil.py", line 414, in _rmtree_safe_fd
    os.rmdir(name, dir_fd=topfd)
OSError: [Errno 16] Device or resource busy: 'archives'
``` 

I guess the relevant part is:

```bash
E: Couldn't find these debs: usrmerge
```